packages/core/src/discovery.js

```diff
@@ -89,7 +89,7 @@ function parseCookies(cookies) {
   // If cookies is collected via SDK
   if (Array.isArray(cookies) && cookies.every(item => typeof item === 'object' && 'name' in item && 'value' in item)) {
     // omit other fields reason sometimes expiry comes as actual date where we expect it to be double
-    return cookies.map(c => ({ name: c.name, value: c.value, secure: c.secure }));
+    return cookies.map(c => ({ name: c.name, value: c.value, secure: c.secure, domain: c.domain }));
```

Comment:
domain will get overridden in page.js
https://github.com/percy/cli/blob/d86b39a87fdacc10398e78d1202781fdbcb1e17b/packages/core/src/page.js#L89